eplmp-server/eplmp-server-ejb/src/main/java/org/polarsys/eplmp/server/NotifierBean.java

```diff
@@ -230,7 +230,7 @@ public void sendWorkspaceDeletionNotification(Account admin, String workspaceId)
 
         try {
             User adminUser = new User(new Workspace(workspaceId), admin);
-            sendMessage(adminUser, "WorkspaceDeletion_title", "WorkspaceDeletion_text", args);
```

Comment:
Changing recipent of notif has nothing to do in this OOME fix.
Is it really required?

Reply:
Changing the recipient of notif fix another exception when searching for the admin mail of a workspace already deleted.
I will use space instead of tabs next time fix.